.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,29 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.0
+    - name: Set up Python
+      uses: actions/setup-python@v4.7.0
+      with:
+        python-version: '3.11'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
+    - name: Build package
+        run: python -m build
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@release/v1@v1.8.10
+        with:
+          password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
Since this is a new PyPI project maybe use the trusted publishing mode directly?

https://github.com/marketplace/actions/pypi-publish#trusted-publishing